app/src/main/java/dev/sebastiano/bundel/preferences/PreferenceStorage.kt

```diff
@@ -0,0 +1,20 @@
+package dev.sebastiano.bundel.preferences
+
+import dev.sebastiano.bundel.preferences.schedule.TimeRangesSchedule
+import dev.sebastiano.bundel.preferences.schedule.WeekDay
+import kotlinx.coroutines.flow.Flow
+
+internal interface PreferenceStorage {
```

Comment:
Checking the code, it looks like the return type is never used.
How about dropping it for all `set...` calls?

Reply:
Yeah we could, let me do it

Reply:
Done :) 20ddc58